services/core/jobs/verifyTransactionStatistics.js

```diff
@@ -0,0 +1,34 @@
+
```

Comment:
Please consider moving the job to `services/core/jobs/transactionStatistics.js`.